app/pages/sections/catalogue-solutions/order-item/controller.js

```diff
@@ -156,7 +177,22 @@ export const validateOrderItemForm = ({ data, selectedPrice }) => {
 export const getSolution = async ({ solutionId, accessToken }) => {
   const endpoint = getEndpoint({ api: 'bapi', endpointLocator: 'getSolution', options: { solutionId } });
   const solutionData = await getData({ endpoint, accessToken, logger });
-  logger.info(`Retrived solution data from BAPI for ${solutionId}`);
+  logger.info(`Retrieved solution data from BAPI for ${solutionId}`);
 
   return solutionData;
 };
+
+export const postSolution = async ({
```

Comment:
If you could rename this to postSolutionOrderItem to make it little more explicit that is what is going on here

Reply:
done